### PR TITLE
Updated lumberjack

### PR DESCRIPTION
**Pull request by markcleonard**
- Changed import.
- No longer use ddLogLevel. (Log level is set `[AFNetworkLumberjackLogger sharedLogger].level = AFLoggerLevelDebug` and also wrapped in `#ifndef RELEASE`)
- Changed to `respondsToSelector ` as AFNetworkActivityLogger did

### CODE REVIEW

File: AFNetworkLumberjackLogger/AFNetworkLumberjackLogger.m [hunk 1]
```diff
@@ -25,21 +25,15 @@
...
 
 
 static NSURLRequest * AFNetworkRequestFromNotification(NSNotification *notification) {
     NSURLRequest *request = nil;
-    if ([[notification object] isKindOfClass:[AFURLConnectionOperation class]]) {
-        request = [(AFURLConnectionOperation *)[notification object] request];
-    } else if ([[notification object] respondsToSelector:@selector(originalRequest)]) {
+    if ([[notification object] respondsToSelector:@selector(originalRequest)]) {
         request = [[notification object] originalRequest];
+    } else if ([[notification object] respondsToSelector:@selector(request)]) {
```
Comment (RomainBoulay) [inline AFNetworkLumberjackLogger/AFNetworkLumberjackLogger.m]: why did you change the order of the ifs?

Comment (markcleonard) [inline AFNetworkLumberjackLogger/AFNetworkLumberjackLogger.m]: I just duplicated it as it was changed in the original repo.

**Comment by RomainBoulay** [comment]
:+1:
